docs/making-yomitan-dictionaries.md

```diff
@@ -0,0 +1,106 @@
+# Making Yomitan Dictionaries　<!-- omit in toc -->
+
+This document provides an overview on how to create your own Yomitan dictionary.
+
+- [Prerequisites](#prerequisites)
+- [General Process](#general-process)
+- [Tools](#tools)
+- [Read the Schemas](#read-the-schemas)
+- [Packaging A Dictionary](#packaging-a-dictionary)
+- [Examples](#examples)
+- [Schema Validation](#schema-validation)
+- [Conjugation](#conjugation)
+- [Tag Categories](#tag-categories)
+
+## Prerequisites
+
+A basic familiarity with JSON and some coding experience in a language of your choice is required.
+
+## General Process
+
+1. Acquire data (from a website, app, dump, etc.)
+2. Parse the data to make sense of it
```

Comment:
"to make sense of it" isnt really needed imo

Reply:
I added this section a while ago after someone asked (imo) pretty obvious questions but I'm thinking maybe it's just not necessary then.